milli/src/update/delete_documents.rs

```diff
@@ -768,82 +896,166 @@ mod tests {
-            {"id":"15","city":"Arras",            "_geo": { "lat": 50.284487528579950, "lng": 2.7637515844478160 } },
-            {"id":"16","city":"Cambrai",          "_geo": { "lat": 50.179340577906700, "lng": 3.2189409952502930 } },
-            {"id":"17","city":"Bapaume",          "_geo": { "lat": 50.111276127236400, "lng": 2.8547894666083120 } },
-            {"id":"18","city":"Amiens",           "_geo": { "lat": 49.931472529669996, "lng": 2.2710499758317080 } },
-            {"id":"19","city":"Compiègne",        "_geo": { "lat": 49.444980887725656, "lng": 2.7913841281529015 } },
-            {"id":"20","city":"Paris",            "_geo": { "lat": 48.902100060895480, "lng": 2.3708400867406930 } }
+            {"id":"1","city":"Lille",             "_geo": { "lat": 50.6299 as f32, "lng": 3.0569 as f32 } },
+            {"id":"2","city":"Mons-en-Barœul",    "_geo": { "lat": 50.6415 as f32, "lng": 3.1106 as f32 } },
+            {"id":"3","city":"Hellemmes",         "_geo": { "lat": 50.6312 as f32, "lng": 3.1106 as f32 } },
+            {"id":"4","city":"Villeneuve-d'Ascq", "_geo": { "lat": 50.6224 as f32, "lng": 3.1476 as f32 } },
+            {"id":"5","city":"Hem",               "_geo": { "lat": 50.6552 as f32, "lng": 3.1897 as f32 } },
+            {"id":"6","city":"Roubaix",           "_geo": { "lat": 50.6924 as f32, "lng": 3.1763 as f32 } },
+            {"id":"7","city":"Tourcoing",         "_geo": { "lat": 50.7263 as f32, "lng": 3.1541 as f32 } },
+            {"id":"8","city":"Mouscron",          "_geo": { "lat": 50.7453 as f32, "lng": 3.2206 as f32 } },
+            {"id":"9","city":"Tournai",           "_geo": { "lat": 50.6053 as f32, "lng": 3.3758 as f32 } },
+            {"id":"10","city":"Ghent",            "_geo": { "lat": 51.0537 as f32, "lng": 3.6957 as f32 } },
+            {"id":"11","city":"Brussels",         "_geo": { "lat": 50.8466 as f32, "lng": 4.3370 as f32 } },
+            {"id":"12","city":"Charleroi",        "_geo": { "lat": 50.4095 as f32, "lng": 4.4347 as f32 } },
+            {"id":"13","city":"Mons",             "_geo": { "lat": 50.4502 as f32, "lng": 3.9623 as f32 } },
+            {"id":"14","city":"Valenciennes",     "_geo": { "lat": 50.3518 as f32, "lng": 3.5326 as f32 } },
+            {"id":"15","city":"Arras",            "_geo": { "lat": 50.2844 as f32, "lng": 2.7637 as f32 } },
+            {"id":"16","city":"Cambrai",          "_geo": { "lat": 50.1793 as f32, "lng": 3.2189 as f32 } },
+            {"id":"17","city":"Bapaume",          "_geo": { "lat": 50.1112 as f32, "lng": 2.8547 as f32 } },
+            {"id":"18","city":"Amiens",           "_geo": { "lat": 49.9314 as f32, "lng": 2.2710 as f32 } },
+            {"id":"19","city":"Compiègne",        "_geo": { "lat": 49.4449 as f32, "lng": 2.7913 as f32 } },
+            {"id":"20","city":"Paris",            "_geo": { "lat": 48.9021 as f32, "lng": 2.3708 as f32 } }
```

Comment:
Also, why are we specifying `as f32` everywhere? Can't we let rust infer whatever he wants?